src/test/test-execute.c

```diff
@@ -924,8 +924,8 @@ int main(int argc, char *argv[]) {
         can_unshare = have_namespaces();
 
         /* It is needed otherwise cgroup creation fails */
-        if (getuid() != 0)
-                return log_tests_skipped("not root");
+        if (getuid() != 0 || have_effective_cap(CAP_SYS_ADMIN) <= 0)
```

Comment:
Why `getuid()` insteado of `geteuid()` here?

Reply:
no idea. always was that way. will change as i guess for the perms the latter matters more